package.json

```diff
@@ -22,26 +28,37 @@
     "test:e2e:grid": "npm run test:server:start && npm run test:e2e:chrome:grid && npm run test:e2e:firefox:grid",
     "test:grid": "wct --config-file ./wct.docker.conf.json && npm run test:e2e:grid",
     "posttest:e2e": "pkill -f http-server",
-    "test:e2e:chrome": "mocha --chrome ",
-    "test:e2e:chrome:grid": "mocha  --chrome --grid",
+    "test:e2e:chrome": "mocha --chrome",
+    "test:e2e:chrome:grid": "mocha --chrome --grid",
     "test:e2e:firefox": "mocha --firefox",
     "test:e2e:firefox:grid": "mocha --firefox --grid",
     "test:e2e:opera": "mocha --opera",
     "test:e2e:safari": "mocha --safari",
     "test:server:start": "http-server &",
     "release": "np",
-    "release:prepare": "npm run release:prepare:build && npm run release:prepare:imports && npm run release:prepare:css && npm run release:prepare:commit",
-    "release:prepare:build": "minify *.src.js > vl-accordion.js",
-    "release:prepare:imports": "replace '../dist/accordion.js' '/node_modules/vl-ui-accordion/dist/accordion.js' vl-accordion.js",
-    "release:prepare:css": "replace '\\.\\./style.css' '/node_modules/vl-ui-accordion/style.css' vl-accordion.js",
-    "release:prepare:commit": "git add -f vl-accordion.js && git commit --amend --no-edit && git pull",
+    "release:prepare": "npm run release:prepare:build && npm run release:prepare:commit",
+    "release:prepare:build": "npm run release:prepare:build:es6 && npm run release:prepare:build:es6:min && npm run release:prepare:build:node",
+    "release:prepare:build:es6": "npm run release:prepare:build:es6:copy && npm run release:prepare:build:es6:replace",
+    "release:prepare:build:es6:copy": "cp -R -f ./src/*.src.js ./vl-button.js",
```

Comment:
hier en in de rest van de npm scripts de file hernoemen

Reply:
Done